src/status_im2/common/bottom_sheet/view.cljs

```diff
@@ -83,6 +83,16 @@
                (and @expanded? (< end-pan-y collapse-threshold))
                (reset! expanded? false))))))))
 
+(defn close-bottom-sheet-fn
+  [on-cancel]
+  (reset! show-bottom-sheet? false)
+  (when (fn? on-cancel) (on-cancel))
+  (timer/set-timeout
+   #(do
```

Comment:
https://github.com/bbatsov/clojure-style-guide#no-multiple-forms-in-function-literals